tests/unit/features/RandomFourierDotFeatures_unittest.cc

```diff
@@ -176,3 +176,30 @@ TEST(RandomFourierDotFeatures, add_to_dense_test)
 
 }
 
+TEST(RandomFourierDotFeatures, gauss_coeff_dim)
+{
+	int32_t num_dims = 50;
+	int32_t num_vecs = 5;
+	int32_t D = 100;
+
+	SGMatrix<int32_t> data(num_dims, num_vecs);
```

Comment:
no need for this data. Just set to ones or zeros

Reply:
I'll instantiate data with random constants as you said before, if that's not a problem?.